src/common/socket.cpp

```diff
@@ -55,6 +55,8 @@ typedef int socklen_t;
 SOCKET sock_arr[FD_SETSIZE];
 int    sock_arr_len = 0;
 
+std::array<std::unique_ptr<socket_data>, FD_SETSIZE> session;
```

Comment:
I'm not sure how we've ever been compiling, with this missing from a compilation unit 👀